Userland/Utilities/strings.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/Stream.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    TRY(Core::System::pledge("stdio rpath"));
+
+    Vector<StringView> paths;
+    size_t minimum_string_length = 4;
+
+    enum StringOffsetFormat {
+        None = 0,
+        Decimal,
+        Octal,
+        Hexadecimal
+    };
+
+    size_t current_file_position;
+    size_t current_path_position = 0;
```

Comment:
Please try to postpone variable declaration as far as possible, it helps with readability.
And you should always define them.